scripts/main.js

```diff
@@ -35,51 +37,76 @@ const questions = [{
 let score = 0;
 let question_index = 0;
 
-const headerContainer = document.querySelector('#header');
-const listContainer = document.querySelector('#list');
-const submitButton = document.querySelector('#submit');
-const picContainer = document.querySelector('#illustration');
+const wrapContainer = document.querySelector('#wrap');
 
-function clearPage(){
-    headerContainer.innerHTML = '';
-    listContainer.innerHTML = '';
-    console.log(picContainer.innerHTML);
-    picContainer.innerHTML = '';
-}
+let quizCard = document.createElement('div');
+quizCard.className="quiz";
 
+let cardImage = document.createElement('img');
+cardImage.className="illustration";
+cardImage.setAttribute("id", "illustration");
 
+let cardBody = document.createElement('div');
+cardBody.className = "quiz-body";
 
-function showQuestion(){
+let cardHeader = document.createElement('div');
+cardHeader.className = "quiz-header";
+cardHeader.setAttribute("id", "header");
+
+let questionText = document.createElement('h2');
+questionText.className = "title";
+
+let submitButton = document.createElement('button');
+submitButton.className = "submit_answer";
+submitButton.setAttribute("id", "submit");
+submitButton.textContent = "Valider";
+
+let answersList = document.createElement('ul');
+answersList.className = "options";
+answersList.setAttribute("id", "list");
+
+wrapContainer.append(quizCard);
+quizCard.append(cardBody);
+cardBody.append(cardHeader);
+quizCard.append(submitButton);
+
+
+function clearPage(){
+    cardHeader.innerHTML = '';
+    answersList.innerHTML = '';
+    cardImage.innerHTML = '';
+}
 
 
-    // Picture
-    const pictureTemplate = `<img class ="illustration" id ="illustration" src = "assets/%illustration%.png">`
-    const pictureHTML = pictureTemplate.replace('%illustration%', 'illustr_'+ question_index);
-    console.log(pictureHTML);
-    picContainer.innerHTML = pictureHTML;
+function showQuestion(){
+    cardImage.src = questions[question_index]['img_link'];

```

Comment:
Better if you initialize a question at the beginning of a function
```
 const question = questions[questions_index]
 cardImage.src = question.imgLink; 
 ...
```

Reply:
Should I git commit --ammend these changes to the current commit, 
or it's better to make a new one, with resolved problems?

Reply:
It'l be nice if you use amend